js/packages/cli/src/gumdrop-cli.ts

```diff
@@ -470,6 +526,42 @@ programCommand('get_contact')
     }
   });
 
+programCommand('fetch_program')
+  .addHelpText(
+    'before',
+    'Utility to fetch the gumdrop program executable data. Useful for doing a diff against a local build. Bytes are formatted for easy comparison to `xxd` output',
+  )
+  .action(async options => {
+    log.info(`Parsed options:`, options);
+
+    const connection = new anchor.web3.Connection(
+      //@ts-ignore
+      options.rpcUrl || anchor.web3.clusterApiUrl(options.env),
+    );
+
+    const programPointer = await connection.getAccountInfo(
+      GUMDROP_DISTRIBUTOR_ID,
+    );
+    const executableBufferKey = new PublicKey(programPointer.data.slice(4));
+
+    const programBuffer = await connection.getAccountInfo(executableBufferKey);
+    const programData = programBuffer.data.slice(0x30 - 3); // by inspection lol...
```

Comment:
lol

Reply:
OK size of https://github.com/solana-labs/solana/blob/master/sdk/program/src/bpf_loader_upgradeable.rs#L41 (4 + 8 + 1 + 32)

Reply:
I'm just amused at the comment haha. "by inspection lol"